test/uk/gov/hmrc/nisp/views/LandingViewSpec.scala

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.nisp.views
+
+import org.scalatest.mock.MockitoSugar
+import play.api.mvc.{AnyContent, Request}
+import play.api.test.FakeRequest
+import play.api.test.Helpers.{contentAsString, _}
```

Comment:
In all your tests you have this import. The first is importing just contentAsString, the `_` imports everything. You dont need the first part
```suggestion
import play.api.test.Helpers._
```

Reply:
You should make this change in all of your test files